backend/src/main/java/site/coduo/retrospect/domain/RetrospectContents.java

```diff
@@ -0,0 +1,53 @@
+package site.coduo.retrospect.domain;
+
+import java.util.List;
+import java.util.stream.IntStream;
+
+import lombok.Getter;
+
+@Getter
+public class RetrospectContents {
+
+    private static final int RETROSPECT_CONTENTS_SIZE = 4;
+
+    private final List<RetrospectContent> values;
+
+    public RetrospectContents(final List<RetrospectContent> values) {
+        validateValues(values);
+        this.values = values;
+    }
+
+    private void validateValues(final List<RetrospectContent> values) {
+        if (values == null) {
+            throw new IllegalArgumentException("회고 문항 내용들로 null을 입력할 수 없습니다.");
+        }
+
+        if (values.size() != RETROSPECT_CONTENTS_SIZE) {
+            throw new IllegalArgumentException("회고 내용 개수는 " + RETROSPECT_CONTENTS_SIZE + "개여야 합니다. - " + values.size());
+        }
+    }
+
+    public static RetrospectContents of(final List<String> answers) {
+        validateAnswers(answers);
+        final List<RetrospectAnswer> retrospectAnswers = answers.stream().map(RetrospectAnswer::new).toList();
```

Comment:
두번째 메서드 체이닝부터 개행을 주면 좋을 것 같아요!
```suggestion
        final List<RetrospectAnswer> retrospectAnswers = answers.stream()
        .map(RetrospectAnswer::new)
        .toList();
```

Reply:
요게 팀 컨벤션 자동 설정 때문에 계속 아래와 같이 자동으로 변하네요.. 에디터를 손봐야하는데 시간이 좀 걸릴거 같아서 이것도 리팩토링PR에서 적용하겠습니다.